mindsdb/integrations/handlers/dremio_handler/requirements.txt

```diff
@@ -0,0 +1,2 @@
+pandas
```

Comment:
This is already depenedency of mindsdb, so we can exclude it from here

Reply:
Got it.